marathon/core/traefik.yml

```diff
@@ -0,0 +1,96 @@
+- vars:
+    traefik_conf_path: "/etc/traefik/traefik.toml"
+    traefik_conf_url: "{{ zk_http_url }}/{{ traefik_conf_path }}"
+
+  block:
+    - name: upload traefik conf in zk
+      znode:
+        hosts: "{{ zk_host }}"
+        name: "{{ traefik_conf_path }}"
+        value: "{{ lookup('template', 'traefik/traefik.toml') }}"
+        state: present
+
+    - name: "Deploy Traefik Reverse Proxy"
+      marathon:
+        marathon_uri: "{{ marathon_uri }}"
+        group_id: traefik/
+        group_json:
+          dependencies:
+            - /core/lb
+            - /core/logging
+          apps:
+            - id: "router"
+              cpus: 2
+              mem: 512
+              disk: 0
+              instances: 2
+              args:
+                - "--logLevel=DEBUG"
+                - "--configfile=/mnt/mesos/sandbox/traefik.toml"
+              container:
+                type: DOCKER
+                docker:
+                  # image: "traefik"
+                  image: stibbons31/traefik-workaround-marathon-bad-gateway:workaround_marathon_bad_gateway
+                  # network: "HOST"
+                  network: "BRIDGE"
+                  privileged: true
+                  parameters: "{{ docker_log_driver_parameters }}"
+                  forcePullImage: false
+                  portMappings:
+                    - containerPort: 80
+                      protocol: tcp
+                      hostPort: 80
```

Comment:
needed to force to take the port 80 (it's it the entrypoint for routing) and have the URL in the marathon ui with the right port.